yeat/tests/test_cli.py

```diff
@@ -56,3 +57,19 @@ def test_duplicate_assembly_algorithms():
     error_message = r"Found duplicate assembly algorithm with `--assemblers` flag: \[\[spades\]\]!"
     with pytest.raises(ValueError, match=error_message):
         yeat.cli.check_assemblers(assemblers)
+
+
+def test_unicycler(capsys, tmp_path):
+    wd = str(tmp_path)
+    arglist = [
+        data_file("short_reads_1.fastq.gz"),
```

Comment:
Just curious: what is the source of these new files?

Reply:
They are from `Unicycler`'s github. https://github.com/rrwick/Unicycler/tree/main/sample_data

These were some of the datasets that I found and Sean referred me to.